app/assets/v2/js/shared.js

```diff
@@ -1412,3 +1412,11 @@ const getVideoMetadata = (videoURL) => {
     'url': videoURL
   };
 };
+
+/**
+ * bootstrap breakpoints
+ */
+const breakpoint_sm = 576;
+const breakpoint_md = 768;
+const breakpoint_lg = 992;
+const breakpoint_xl = 1200;
```

Comment:
Maybe you want to get it from the css variables
getComputedStyle(document.documentElement).getPropertyValue('--breakpoint-sm');
--breakpoint-xs: 0;
    --breakpoint-sm: 576px;
    --breakpoint-md: 768px;
    --breakpoint-lg: 992px;
    --breakpoint-xl: 1200px;
    
    But just a suggestion as probably could delay the nav to be ready

Reply:
Good idea 👍 I'm already reading `--gc-menu-transition-duration` from the styles so it shouldn't introduce any further delay...

I've bound the whole of `navbar.js` to `DOMContentLoaded` and added `showMenu()` to the `click` event - those changes are now on staging, it seems to work well but let me know if you think the delay is too much and I can switch it back. 7cd0d2e3bf96cc58f09dafa0833650d0731f2c7b